RoktDemo/Resources/SceneDelegate.swift

```diff
@@ -64,6 +77,14 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // to restore the scene back to its current state.
     }
 
+    private func handleURLContexts(_ urlContexts: Set<UIOpenURLContext>) {
+        if let urlContext = urlContexts.first {
+            let url = urlContext.url
+            let queryItems = URLComponents(url: url, resolvingAgainstBaseURL: false)?.queryItems
+            let config = queryItems?.first(where: { $0.name == "config" })?.value
```

Comment:
Do we want to validate the config a bit here?

Reply:
IMO that can be done on the layout preview page. The URL handler simply get the config query param and pass it to the relevant page that can handle it. That page would know the domain model and the validations needed.